src/events/events.gateway.ts

```diff
@@ -46,8 +47,11 @@ export class EventsGateway implements OnGatewayConnection, OnGatewayDisconnect {
   // NOTE events for Game below:
 
   @SubscribeMessage('waiting')
-  handleWaiting(@ConnectedSocket() client: Socket): void {
-    return this.eventsService.handleWaiting(this.server, client);
+  handleWaiting(
+    @ConnectedSocket() client: Socket,
+    @MessageBody() { type, mode }: OnMatchTypeModeDto,
```

Comment:
현재 방식대로는 값이 넘어오지 않습니다.

아래와 같이 수정해야 값이 넘어와용

```suggestion
    @MessageBody() [ type, mode ]: string[],
```

Reply:
제가 잘못테스트했습니다. 값이 잘 넘어와요!